nikola/plugin_categories.py

```diff
@@ -588,6 +610,7 @@ class Taxonomy(BasePlugin):
     generate_atom_feeds_for_post_lists = False
     template_for_single_list = "tagindex.tmpl"
     template_for_classification_overview = "list.tmpl"
+    add_postcount_in_overview = False
```

Comment:
I don't think so. We can also make it `True` and don't allow to set it to `False` (i.e. always insert the variables).